doc/changelog.md

```diff
@@ -9,6 +9,10 @@
   value method using the native Kokkos kernel for the Lightning-Kokkos device.
   [#70](https://github.com/PennyLaneAI/catalyst/pull/70)
 
+* Allow `catalyst.grad` to be used on any traceable function (within a qjit context).

```

Comment:
Don't forget to add your name to the list of contributors :slightly_smiling_face:

Reply:
Thanks :D